.github/pull_request_template.md

```diff
@@ -1,46 +1,87 @@
-## Cover letter
+<!--
+
+See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md##pull-request-body
+for more details and examples of what is expected in a PR body.
+
+-->
 
-Describe in plain language the motivation (bug, feature, etc.) behind the change in this PR and how the included commits address it.
+## Cover Letter
 
-<!-- Use the GitHub keyword `Fixes` to link to bug(s) this PR will fix. -->
-Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...
+<!--
+
+Content in this section is REQUIRED.
 
-## Backport Required
+Describe, in plain language, the motivation behind the change (bug fix,
+feature, improvement) in this PR and how the included commits address it.
 
-<!-- Specify which branches this should be backported to, e.g.: -->
-- [ ] not a bug fix
-- [ ] issue does not exist in previous branches
-- [ ] papercut/not impactful enough to backport
-- [ ] v22.2.x
-- [ ] v22.1.x
-- [ ] v21.11.x
+Add the GitHub keyword `Fixes` to link to bug(s) this PR will fix, e.g.
 
-## UX changes
+  Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...
 
-Describe in plain language how this PR affects an end-user. What topic flags, configuration flags, command line flags, deprecation policies etc are added/changed.
+If this PR is a backport, link to the original with `Backport of PR`, e.g.
 
-<!-- don't ship user breaking changes. Ping PMs for help with user visible changes  -->
+  Backport of PR #PR-NUMBER
+
+-->
+
+## Backports Required
 
-## Release notes
 <!--
 
-If this PR does not need to be included in the release notes, then
-simply have a bullet point for `none` directly under the `Release notes`
-section, e.g.
+Content in this section is REQUIRED if this is a PR to `dev` branch.
+
+If this PR needs backport, then list the branches to backport to, e.g.
+
+  - [ ] v22.2.x
+  - [ ] v22.1.x
+  - [ ] v21.11.x
+
+If this PR does not need backport, then list `none`, e.g.
+
+  * none
```

Comment:
I really like having people click a reason for why they weren't backporting.

To make it easier to parse we could just prefix the checkboxes that aren't a git tag with "none" or somesuch.

Reply:
i've been interpreting the checkboxes as items in a todo lists because checkboxes are summarized as a blue progress circle indicating the number of tasks completed on the [list of PRs page](https://github.com/redpanda-data/redpanda/pulls). so, i've avoided giving a list of checkboxes as a the options for the author to choose from for "none".

how about updating the instructions to say something like `If this PR does not need backport, then list "none" with the reason` and giving examples like:

```suggestion
  * none - not a bug fix
or
  * none - issue does not exist in previous branches
or
  * none - papercut/not impactful enough to backport
```

Reply:
I dislike having to block select text and remove it either side of the option I want to select, rather than just putting an X in the box.

It's annoying that github insists on treating all checkboxes as todo items, but i don't think it's actually hurt us up to this point

Reply:
i've restored the original list of checkboxes using `none` as the prefix and updated the `CONTRIBUTING.md` text